packages/docs/src/page-configs/ui-elements/input/examples/Default.vue

```diff
@@ -43,6 +43,36 @@
       placeholder="Readonly"
       readonly
     />
+    <va-input
+      v-model="value"
+      counter
+      class="mb-4"
+    />
+    <va-input
+      v-model="value"
+      :max-length="30"
+      counter
+      class="mb-4"
+    >
+      <template #counter="{ counter, maxLength }">
+        <span>Remaining symbols</span>
```

Comment:
I'm not sure you'll have a case with "Remaining symbols" text. Counter is a wrong slot for this amount of text.

Maybe you can do something like "30 characters left".